packages/experience-tool-kit/src/client/api.ts

```diff
@@ -0,0 +1,210 @@
+// Interaction requests
+
+import {
+  CreateExperienceApiPayload,
+  IdentificationApiPayload,
+  PasswordVerificationPayload,
+  VerificationCodeIdentifier,
+  InteractionEvent,
+  UpdateProfileApiPayload,
+  MfaFactor,
+} from '@logto/schemas';
+import ky from 'ky';
+
+/**
+ * Returns the redirect URL if the interaction event is SignIn or Register.
+ * Returns undefined if the interaction event is ForgotPassword.
+ */
+export type SubmitInteractionResponse = {
+  redirectUrl?: string;
+};
+
+const experienceApiPrefix = '/api/experience';
+
+const experienceApiRoutes = Object.freeze({
+  prefix: experienceApiPrefix,
+  identification: `${experienceApiPrefix}/identification`,
+  verification: `${experienceApiPrefix}/verification`,
+  submit: `${experienceApiPrefix}/submit`,
+  profile: `${experienceApiPrefix}/profile`,
+  mfa: `${experienceApiPrefix}/profile/mfa`,
+});
+
+const api = ky;
+
+export function initInteraction(payload: CreateExperienceApiPayload) {
+  return api
+    .put(experienceApiRoutes.prefix, {
+      json: payload,
+    })
+    .json();
+}
+
+export function identifyUser(payload: IdentificationApiPayload) {
+  return api
+    .post(experienceApiRoutes.identification, {
+      json: payload,
+    })
+    .json();
+}
```

Comment:
if we have swagger we can actually generate an api sdk with native `fetch`. no need for ky. i found and tried this one which looks good: https://github.com/acacode/swagger-typescript-api